packages/protocol/test/genesis/test_config.json

```diff
@@ -1,5 +1,6 @@
 {
   "contractOwner": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "contractAdmin": "0xc0ffee254729296a45a3885639AC7E10F9d54979",
```

Comment:
this is a random address, not sure if it matters here

Reply:
this json is only for testing, so i think its ok.